projects/solar/src/solar_config.c

```diff
@@ -24,6 +24,16 @@
 // Must be calibrated.
 #define SOLAR_MPPT_VIN_SCALING_FACTOR 1.0f
 
+// Overcurrent threshold for the output current of the array. 9A.
+#define SOLAR_OUTPUT_OVERCURRENT_THRESHOLD_uA 9000000
+
+// Overvoltage threshold for the sum of the output voltages of the MPPTs. 160V.
+// Note that this sums 5 MPPTs' output voltages on the 5 MPPT board and 6 on the 6 MPPT board.
+#define SOLAR_OUTPUT_OVERVOLTAGE_THRESHOLD_mV 160000
+
+// Overtemperature threshold for any individual thermistor. To be calibrated, currently 100C.
+#define SOLAR_OVERTEMPERATURE_THRESHOLD_dC 1000
+
```

Comment:
That would make sense if we wanted to specify the threshold with respect to the ADC reading rather than the actual value. Probably something to do later. (Though I believe we won't have a scaling factor for the case of the temperature, instead hardware will (eventually) send me the curve for converting ADC readings from the thermistors to temperatures.)